libtcmu.c

```diff
@@ -390,6 +390,11 @@ void tcmu_block_device(struct tcmu_device *dev)
 {
 	int rc;
 
+	if (!tcmu_is_ring_reset_support(dev)) {
+		tcmu_dev_warn(dev, "reset ring is not supported!\n");
```

Comment:
Let's make this more descriptive like below:

 "Kernel does not support the block_dev action.\n"

You can then also kill the explicit check for ENOENT since we already check it above now.

Reply:
sure